frontend/src/assets/css/variables.scss

```diff
@@ -38,9 +38,6 @@
     // Porcelain
     --porcelain: #f5f6f7;
 
-    // Sunglow
-    --sunglow: #ffd231;
-
```

Comment:
We don`t already have duplication error with this variable, so no need to remove it

Reply:
Fixed as merge conflict